spec/features/idv/in_person_spec.rb

```diff
@@ -618,32 +469,22 @@
       complete_location_step(user)
     end
 
-    it 'can redo the address page form after it is skipped' do
-      complete_state_id_step(user, same_address_as_id: true)
-      # skip address step
-      complete_ssn_step(user)
-      # click update address button on the verify page
-      click_button t('idv.buttons.change_address_label')
-      expect(page).to have_content(t('in_person_proofing.headings.update_address'))
-      fill_out_address_form_ok(same_address_as_id: true)
-      click_button t('forms.buttons.submit.update')
-      expect(page).to have_content(t('headings.verify'))
-      expect(page).to have_current_path(idv_in_person_verify_info_path)
-    end
-
     it 'allows user to update their residential address as different from their state id' do
       complete_state_id_step(user, same_address_as_id: true)
+      # skip address step b/c residential address is same as state id address
       complete_ssn_step(user)
 
-      # click "update residential address"
+      # click update residential address
       click_button t('idv.buttons.change_address_label')
       expect(page).to have_content(t('in_person_proofing.headings.update_address'))
-      # change something in the address
+      # expect page to have fields already populated
```

Comment:
I see the comment about having fields already populated but no test to prove it. I don't think you need a test for every field but maybe just one of the required fields is enough. What do you think?

Reply:
oh yes thank you for catching this comment. I actually ended up adding the tests that checked for that in the state_id and address feature specs directly. I'll delete this comment.

Reply:
actually, i still will add one here to show that when same address as id was true that it has these fields populated